backend/kernelCI_app/management/commands/process_pending_aggregations.py

```diff
@@ -0,0 +1,394 @@
+import hashlib
+import time
+from typing import Sequence
+from django.core.management.base import BaseCommand
+from django.db import connection, transaction
+from kernelCI_app.constants.general import MAESTRO_DUMMY_BUILD_PREFIX
+from kernelCI_app.models import (
+    Builds,
+    Checkouts,
+    HardwareStatusEntityType,
+    PendingTest,
+    ProcessedHardwareStatus,
+    SimplifiedStatusChoices,
+    StatusChoices,
+)
+
+
+def get_hardware_key(origin: str, platform: str, checkout_id: str) -> bytes:
+    return hashlib.sha256(f"{origin}|{platform}|{checkout_id}".encode("utf-8")).digest()
+
+
+def calculate_status_count(status: str) -> tuple[int, int, int]:
+    if status in [StatusChoices.PASS, SimplifiedStatusChoices.PASS]:
+        return (1, 0, 0)
+    elif status in [StatusChoices.FAIL, SimplifiedStatusChoices.FAIL]:
+        return (0, 1, 0)
+    else:
+        return (0, 0, 1)
```

Comment:
stick with SimplifiedStatusChoices, with that you can create a map which you can use directly as `simplified_status_choices_count[status]`

```py
simplified_status_choices_count = {
    SimplifiedStatusChoices.PASS: (1, 0, 0),
    SimplifiedStatusChoices.FAIL: (0, 1, 0),
    SimplifiedStatusChoices.INCOMPLETE: (0, 0, 1),
}
```